### ISSUE TITLE
Vue 3 / CKEditor 5 / Webview Android App > When type anyword and click 'Space' it duplicates the word

### ISSUE THREAD

**Issue by greeto** [ckeditor/ckeditor5#17699]
## 📝 Provide detailed reproduction steps (if any)

1. Create WebView android app.
2. Create vue 3 app and install any CKEditor 5 
3. From the Android Webivew > Type and word, for example when typing "hello" and click 'space' you see inside the editor "hello hello"

### ✔️ Expected result

_should be just 1 word ."hello"

### ❌ Actual result

_got 2 words: "hello hello"

### ❓ Possible solution

_I don't know, but it works fine with TinyMCE.

## 📃 Other details

* Browser: Android webivew
* OS: Any.
* First affected CKEditor version: 5
* Installed CKEditor plugins: none

Image: https://postimg.cc/jLXG7pf5

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Witoso** [comment]
Could you specify the version of the CKEditor 5 you are using?

**Comment by Witoso** [comment]
Hmm, we fixed similar problems last year, could you share as well the Android version, phone type, and ideally the keyboard type you are using? Thanks!

**Comment by greeto** [comment]
Sure thing!

**Devices with Problem**(at least): Google Pixel 3a | Galaxy S23 Ultra | Huawei P30 Pro
**Android version**: 15
**Keyboard type**: The Default | Happens also on Microsoft Keyboard.

**Screen Recording Example Of the Problem:**
https://streamable.com/ax4k55

(I got this problem from multiple clients(devices) so it's not one specific device issue)

Very urgent for us :/

**Comment by greeto** [comment]
Any news/ideas maybe? becomes really urgent for us :/ 

@Witoso

**Comment by Witoso** [comment]
Hey, we usually don't hastily improve the typing pipeline, this is a very complex matter. We haven't got more requests like that, so we need to pinpoint the cause, and we will be investigating that.

If you're a premium customer, please contact our [support](https://ckeditor.com/contact/) to connect this issue to your account.

**Comment by greeto** [comment]
Hi, I'm not a premium customer, but if I will be, will it make it faster?
Also, how do I become a premium? @Witoso

**Comment by Witoso** [comment]
So far we couldn't reproduce the issue.

Steps we took:

1. Use the [WebView Android app](https://play.google.com/store/apps/details?id=com.snc.test.webview2&hl=en).
2. Set up a Vue 3 app and create a CKEditor 5.
3. Visit the page in the WebView.

We were unable to replicate the issue that was reported with duplicated typing. Could you share some reproduction methods we can use?

**Comment by greeto** [comment]
Hi,
1. Can you share the Webview code you did please?
2. Here's the base of the Webview I'm using: https://pastebin.com/g4QJLTez
3. Android Device Emulator: Pixel 3a API 32 arm64-v8a (Android 12)

@Witoso

**Comment by Witoso** [comment]
We didn't create a webview, we are not experts in mobile development :). We used a proxy app that you can test as well ([WebView Android app](https://play.google.com/store/apps/details?id=com.snc.test.webview2&hl=en)).

**Comment by greeto** [comment]
Thanks.

I've opened some debug logs and now I see some errors related to CkEditor while I'm typing:

https://i.postimg.cc/90f1b168/ss.png

@Witoso

**Comment by greeto** [comment]
Ok I got the error solved somehow!
Seems I used some custom 'user agent' on that webview but it caused CKEditor throwing errors.
So I've disabled that now and it got resolved.

**Comment by Witoso** [comment]
Good catch, based on the environment the typing may behave differently as devices' input methods are not standardized. AFAIK, we use user agents to detect the environments.
